File: TransCoder_tokenized_test_set_functions/java/FIND_PAIRS_IN_ARRAY_WHOSE_SUMS_ALREADY_EXIST_IN_ARRAY.java
static void findPair ( int [ ] arr, int n ) {
  boolean found = false;
  for ( int i = 0;
  i < n;
  i ++ ) {
    for ( int j = i + 1;
    j < n;
    j ++ ) {
      for ( int k = 0;
      k < n;
      k ++ ) {
        if ( arr [ i ] + arr [ j ] == arr [ k ] ) {
          System . out . println ( arr [ i ] + " " + arr [ j ] );
          found = true;
        }
      }
    }
  }
  if ( found == false ) System . out . println ( "Not exist" );
}

File: TransCoder_tokenized_test_set_functions/python/FIND_PAIRS_IN_ARRAY_WHOSE_SUMS_ALREADY_EXIST_IN_ARRAY.py
def findPair ( arr, n ) :
    found = False
    for i in range ( 0, n ) :
        for j in range ( i + 1, n ) :
            for k in range ( 0, n ) :
                if ( arr [ i ] + arr [ j ] == arr [ k ] ) :
                    print ( arr [ i ], arr [ j ] )
                    found = True
    if ( found == False ) :
        print ( "Not exist" )
    
